test/unit/alignment/configuration/align_config_common_test.cpp

```diff
@@ -53,7 +53,7 @@ TEST(alignment_configuration_test, symmetric_configuration)
 TEST(alignment_configuration_test, number_of_configs)
 {
     // NOTE(rrahn): You must update this test if you add a new value to seqan3::align_cfg::id
-    EXPECT_EQ(static_cast<uint8_t>(seqan3::detail::align_config_id::SIZE), 13);
+    EXPECT_EQ(static_cast<uint8_t>(seqan3::detail::align_config_id::SIZE), 19);
```

Comment:
When does this happen? Is this part of this PR?

Reply:
This happens in the first 5 commits. Each output option is a separate commit and updates this value.